docs/bin/bundle.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+#
+# Bundle a sample project for download.
+#
+# Usage:
+#   bundle.sh --zip <file> <dir>
+
+set -euo pipefail
+
+function _script_path {
+  local source="${BASH_SOURCE[0]}"
+  while [ -h "$source" ] ; do
+    local linked="$(readlink "$source")"
+    local dir="$(cd -P $(dirname "$source") && cd -P $(dirname "$linked") && pwd)"
+    source="$dir/$(basename "$linked")"
+  done
+  echo ${source}
+}
+
+readonly script_path=$(_script_path)
+readonly script_dir="$(cd -P "$(dirname "$script_path")" && pwd)"
+readonly docs_dir="$(cd "$script_dir/.." && pwd)"
+readonly bundle_dir="$docs_dir/build/bundle"
+
+function _remove_doc_tags {
+  local -r dir="$1"
+  # note: use commands that are compatible with both GNU sed and BSD (macOS) sed
+  find "$dir" -type f -exec sed -i.bak "/tag::[^\[]*\[.*\]/d" {} \; -exec rm -f {}.bak \;
+  find "$dir" -type f -exec sed -i.bak "/end::[^\[]*\[.*\]/d" {} \; -exec rm -f {}.bak \;
```

Comment:
Are there any callouts in the shopping cart samples being bundled? If there are, then we need some sed to remove those too.

Reply:
I'm not sure I know what you mean with callouts?

Reply:
https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/

Any code blocks that have the numbered callouts with `// <1>` or similar

Reply:
There are some in the customer registry samples

Reply:
Ah thanks :) I've added a regex to do that.